modules/nf-core/ale/main.nf

```diff
@@ -0,0 +1,51 @@
+process ALE {
+    tag "$meta.id"
+    label 'process_single'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ale:20180904--py27ha92aebf_0':
+        'biocontainers/ale:20180904--py27ha92aebf_0' }"
+
+    input:
+    tuple val(meta), path(asm)
+    tuple val(meta1), path(bam)
```

Comment:
Can this meta ever be different? If not, it can just be another val(meta)

Reply:
Thanks for your comments and tips. I have followed your suggestion and merged the input into a single tuple.